### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by Robertleoj** [msminhas93/DeepLabv3FineTuning#23]
Why do you use MSELoss? Shouldn't cross-entropy loss be used for semantic segmentation?
